test/unit/legacy-data.html

```diff
@@ -0,0 +1,229 @@
+<!doctype html>
+<!--
+@license
+Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<html>
+<head>
+  <meta charset="utf-8">
+  <script src="../../node_modules/@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
+  <script src="wct-browser-config.js"></script>
+  <script src="../../node_modules/wct-browser-legacy/browser.js"></script>
+  <!-- Applies LegacyDataMixin to all legacy Polymer({..}) elements -->
+  <script type="module" src="../../lib/legacy/legacy-data-mixin.js"></script>
+</head>
+<body>
+
+  <dom-module id="x-data">
+    <script type="module">
+      import {Polymer} from '../../polymer-legacy.js';
+      Polymer({
+        is: 'x-data',
+        properties: {
+          singleProp: String,
+          multiProp1: String,
+          multiProp2: String
+        },
+        _legacyUndefinedCheck: true,
+        observers: [
+          'staticObserver("staticObserver")',
```

Comment:
Shouldn't we also test inline template expressions that have multiple arguments?

Reply:
Added.